addon/services/rollbar.js

```diff
@@ -10,11 +10,17 @@ export default Ember.Service.extend({
   }),
 
   config: Ember.computed(function() {
-    return Ember.getOwner(this).resolveRegistration('config:environment').emberRollbarClient;
+    const applicationConfig = Ember.getOwner(this).resolveRegistration('config:environment');
+    const code_version = applicationConfig.APP.version;
+    const userConfig = applicationConfig.emberRollbarClient;
+    if ( !userConfig.payload.client.javascript.code_version ) {
+      userConfig.payload.client.javascript.code_version = code_version;
+    }
+    return userConfig;
   }),
 
   rollbarClient(customConfig = {}) {
-    let config = Ember.assign(this.get('config'), customConfig);
+    const config = Ember.assign(this.get('config'), customConfig);
```

Comment:
https://madhatted.com/2016/1/25/let-it-be

Reply:
Can we replace assign with lodash deepMerge?
Here is the example: https://github.com/Exelord/ember-rollbar-client/pull/4/files#diff-b3e6cb7ec4b284436511fef3c2eb318a

Reply:
Should I apply the changes that will be probably introduced by PR that will be merged later? I don't wanna copy all the improvements from there since I want only to focus on actual new behaviour rather than code improvements/bug fixes already handled in another PR.

Reply:
Yea, I understand. We should not fix it but I can't release a new version with this serious bug :/

Reply:
I've changed it in the later commits.